### ISSUE TITLE
Better wording for login warning

### ISSUE THREAD

**Issue by dik23** [nextcloud/bruteforcesettings#501]
This isn't a great way to describe what a user is seeing.  Even people on my side of the fence are unsure what it means.  An update to make it clearer would be great

![Unsaved Image 1](https://github.com/nextcloud/bruteforcesettings/assets/15139568/37bcbe4e-f0c1-42cc-920f-2911e8d29052)

**Comment by marcoambrosini** [comment]
cc @nextcloud/designers 

How about just "Try again in 30 seconds"?

**Comment by nimishavijay** [comment]
Agreed! Do we know if it's because of any particular reason? Eg wrong email/password too many times, etc? We could also include something like "You have tried to log in too many times with a wrong account name, email or password. Try again in 30 seconds." (but that's still a nice-to-have, just "Try again in 30 seconds" sounds good to me)

**Comment by joshtrichards** [comment]
Is the bottom line here that the message basically gives out *too much* information so it feels like overload for a typical user?

I can see the elegance of just saying "Try again in 30 seconds" because the typical user probably doesn't care about the details (the *why* basically) - they just want to have their expectations sufficiently enough that they know what to do next.

Other than that, the original (existing) error seems accurate unless I'm missing something. (Though I'd change "IP" to "IP address").

If someone wants to clean up the message (which only kicks in after the delay has reached 5s), it's found here:

https://github.com/nextcloud/server/blob/1bc8129623d15b369a7b6bf7ac65931b0e83455e/core/src/components/login/LoginForm.vue#L185
